workspace/__lib__/xod/color/color/patch.cpp

```diff
@@ -2,9 +2,7 @@
 struct State {
 };
 
-struct Type {
-    uint8_t r, g, b;
-};
+using Type = xod::XColor;
```

Comment:
You’re already in the `xod` namespace.

```suggestion
using Type = XColor;
```